src/praisonai/praisonai/cli/commands/managed.py

```
@@ -316,6 +316,31 @@ def sessions_resume(
+    Example:
+        praisonai managed sessions delete sesn_01AbCdEf
+        praisonai managed sessions delete sesn_01AbCdEf --yes
+    """
+    if not confirm:
+        typer.confirm(f"Delete session {session_id}? This cannot be undone.", abort=True)
+    
+    try:
+        client = _get_client()
+        # Note: Anthropic API may not have delete endpoint yet
+        # This is a placeholder for when it becomes available
+        typer.echo(f"Session deletion not yet supported by Anthropic API")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)